researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -50,7 +51,9 @@
 * This is a very simple implementation that hits only part of the Sage Bridge REST API
 * a complete port of the Sage Bridge Java SDK for android: https://github.com/Sage-Bionetworks/BridgeJavaSDK
  */
-public abstract class BridgeDataProvider extends DataProvider {
+@AnyThread
+@Deprecated
```

Comment:
I've basically given up on maintaining backwards compatibility :)

I did this to have a clean BridgeDataProvider file and intend that we replace the old one rather than have one extend the other